pages/common/type.md

```diff
@@ -14,3 +14,7 @@
 - Display the name of the disk file that would be executed:
 
 `type -p {{command}}`
+
+- Output a single word saying if a command is an alias/keyword/function/builtin/file (works only in Bash shell):
```

Comment:
```suggestion
- Display the type of a specific command, alias/keyword/function/builtin/file (Note: works only in Bash shell):
```
- I think "display" is a bit more common accros the pages.
- If we give all possible values of a  command we don't need to say it's a single word.

Reply:
> Furthermore, AFAIK, the latest release of the POSIX standard does not list any options for the command `type` [^1].
> 
> [^1]: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/type.html

This is correct indeed.

> Should pages have examples with unportable options and/or arguments even if the example description explicitly states in which scenarios it will work (e.g. `bash`)?

My opinion: why not, if it explicitly states this? This works on any OS that supports bash.

I support adding these additional options especially because the POSIX version has no options

Reply:
What about if the example description does not explicitly mention the unportable command line arguments or features?

Reply:
> What about if the example description does not explicitly mention the unportable command line arguments or features?

What do you mean with "not explicitly", implicitly? Or removing it? Because in this case I don't see any benefits on removing it, it will not serve for putting more examples.

Also, it's unportable in the sense o f the shell, but it works on all major OSes (Linux, macOS, BSDs and even Windows with e.g. mingw).

I still don't understand why you see explicitly mentioning it as a problem, because it makes things clear, given that, for example: ksh is the default OpenBSD shell and Zsh is the default macOS shell.

> Also, when you replied did you use footnote syntax? If not, carrying the footnotes could be unexpected behaviour from the GitHub mobile app.

Yeah, probably unexpected behavior from it, it wasn't my intention.

Reply:
I did not express myself clearly, my issue was with example descriptions which did not specifically state when a feature will work. The new addition in the page does say so.

Your points are valid and now that I think about it, examples with unportable features can be useful and worth showing.